app/models/application.rb

```diff
@@ -91,6 +30,11 @@ def self.data_label(key)
     ApplicationDraft.human_attribute_name(key)
   end
 
+  def data
+    # require 'pry'; binding.pry
+    ApplicationData.new(self.application_data)
```

Comment:
This is personal preference, but I'd skip the `self` here. Being an AR model, there's a accessor for the attribute and it does not seem to be a pattern to use `self` in this app *(at least from what I see in this PR)* 😉